.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: LNbits CI
+on: [push, pull_request]
+
+jobs:
+
+  lint:
+    uses: ./.github/workflows/lint.yml
+
+  tests:
+    needs: [ lint ]
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
+        db-url: ["", "postgres://lnbits:lnbits@0.0.0.0:5432/lnbits"]
```

Comment:
looks a bit weird with the db-url in the workflow name instead of `postgres` but not that big of a deal